contracts/consensus/Consensus.sol

```diff
@@ -914,15 +914,30 @@ contract Consensus is MasterCopyNonUpgradable, CoreLifetimeEnum, MosaicVersion,
         anchor_ = address(anchor);
         metachainId_ = hashMetachainId(anchor_);
 
+        uint256 genesisMetablockHeight = uint256(0);
+
+        Metablock storage genesisMetablock = metablockchains[metachainId_][genesisMetablockHeight];
+        genesisMetablock.metablockHash = keccak256(
+            abi.encodePacked(
+                "Mosaic-1 Testnet Gen-1 Genesis Metablock hash",
```

Comment:
Do we plan to keep it like this?
If yes then instead of `Testnet` can we use the `chainid`? We already have the `getChainId` function.

Reply:
I am not really sure, this has been added by Ben.
We can either create a follow up ticket or ask Ben's input.
/cc @benjaminbollen

Reply:
Sure